src/commons/eventsourcing/store.rs

```diff
@@ -178,152 +92,94 @@ where
     pub fn warm_aggregate(&self, handle: &MyHandle) -> StoreResult<()> {
```

Comment:
Is the fn comment about "surplus entries will be archived" still accurate given the code removed and changes made to `fn recover()` ?

Reply:
Ah I see in #1078 you haven't yet ticked the `Fill cache on warmup` box, so maybe that relates to this.